gen/enum_test.go

```diff
@@ -25,11 +25,12 @@ import (
 	"fmt"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
think imports were grouped in this repo before -- can we keep the third party imports at the bottom?

Reply:
ops, ide linter didnt' capture this